cdap-ui/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "cdap-ui",
-  "version": "0.1.3",
+  "version": "0.2.8",
```

Comment:
as of now this version number in package.json does nothing, apart from being shown in the console when the app boots.